README.md

```diff
@@ -120,6 +123,30 @@ need to be provided in order for this action to successfully run using one of th
 | ------------------------- | -------- | ----------- |
 | `imageToDeploy`           | Yes (for this scenario) | The name of the image that has already been pushed to a registry and will be deployed to the Container App by this action. If this image is found in an ACR instance that requires authentication to pull, the `acrName` argument, or the `acrUsername` and `acrPassword` arguments, can be provided to authenticate requests to the ACR instance. |
 
+### Arguments required for using a YAML configuration file
+
+| Argument name             | Required | Description |
+| ------------------------- | -------- | ----------- |
+| `yamlConfigPath`          | Yes (for this scenario) | Full path (on the executing GitHub runner) to the YAML file detailing the configuration of the Container App. |
+
+#### Important note on the YAML configuration file
+
+When providing the `yamlConfigPath`, identity arguments (_e.g._, `containerAppName`, `resourceGroup`) provided
+to this action will still be used, but non-identity arguments (_e.g._, `containerAppEnvironment`, `environmentVariables`,

```

Comment:
Hmm, this almost seems like it contradicts the yaml parameter explanation in https://learn.microsoft.com/en-us/cli/azure/containerapp?view=azure-cli-latest&preserve-view=true#az-containerapp-create-optional-parameters, which states that `All other parameters will be ignored.`. Is it just the required parameters that are not ignored? Specifically, the `containerAppName`, `resourceGroup` you have listed above?

Reply:
I thought the same thing too, but the CLI behavior is a little odd -- the `--resource-group` and `--name` arguments are _required_, independent of if you've provided the `--yaml` argument.

Whatever value you provide for `--resource-group` is going to be the resource group that the Container App is created in (or that it lives in); however, whatever value you provide for `--name` is overridden by the `name` property in the YAML configuration file, so that statement they make is _almost_ true. I'll this piece of documentation to call out that only the `resourceGroup` argument for our action is necessary.